config/opensearch_mappings.json

```diff
@@ -315,8 +315,28 @@
           }
         }
       },
-      "publication_information": {
-        "type": "text"
+      "publishers": {
+        "type": "nested",
+        "include_in_parent": "true",
+        "properties": {
+          "name": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "normalizer": "lowercase"
+              }
+            }
+          },
+          "date": {
+            "type": "keyword",
+            "normalizer": "lowercase"
+          },
```

Comment:
I think this feels right, that we don't also try and parse this to a date string.  Maybe we'll revisit if we extend this pattern a bit, but if we know we're already duplicating dates to `Dates` then this as a date field likely isn't needed.  With, the benefit of handling non-normal date strings that may still contain information.